beacon-chain/core/blocks/block_operations.go

```diff
@@ -783,37 +782,17 @@ func ProcessAttestationNoVerify(
 //        return False
 //    return True
 func VerifyIndexedAttestation(ctx context.Context, beaconState *stateTrie.BeaconState, indexedAtt *ethpb.IndexedAttestation) error {
-	ctx, span := trace.StartSpan(ctx, "core.VerifyIndexedAttestation")
+	ctx, span := trace.StartSpan(ctx, "core.VerifyIndexedAttestationSig")
```

Comment:
```suggestion
	ctx, span := trace.StartSpan(ctx, "core.VerifyIndexedAttestation")
```
Either the function's name should change to `VerifyIndexedAttestationSig`, or `StartSpan` comment left intact.

Reply:
thanks. was auto replaced somehow. good catch